### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by unphased** [rui314/mold#870] [General]
Hey, I just had a hack session and enabled this linker in my project at work. The increase in speed is INCREDIBLE.

The only snag I ran into was I had to add a `-L /usr/local/lib` to the gcc flags as well to make mold search for a library there, which gcc originally didn't need. 

This thing is AMAZING. I've been waiting over a decade since I started C++ programming for a faster linker. I knew about gold and lld, but nothing until now had stood out as something seemingly worth the added complexity to the build system. And the benefits go far beyond just C++. You have singlehandedly brought us into a new age of compiled language developer experience. I can't think of enough good things to say about this.

**Comment by rui314** [comment]
Thank you for your message, @unphased! I'm pleased that our program works for you well.

**Comment by unphased** [comment]
I wish more projects took the new trends in hardware seriously like you've done here. 
The number of cores in computers is only going to increase from here on out, the writing's been on the wall for 10 years.
